native/src/routes/feedback/containers/FeedbackModalContainer.js

```diff
@@ -1,81 +1,21 @@
-import { CityModel } from 'api-client'
+import type { FeedbackModalRouteType, OffersRouteType } from 'api-client'
 import type { NavigationPropType, RoutePropType } from '../../../modules/app/constants/NavigationTypes'
-import type { StoreActionType } from '../../../modules/app/StoreActionType'
-import type { StateType } from '../../../modules/app/StateType'
-import createNavigateToFeedbackModal from '../../../modules/navigation/createNavigateToFeedbackModal'
-import type { FeedbackOriginType } from '../../../modules/feedback/FeedbackContainer'
 import FeedbackContainer from '../../../modules/feedback/FeedbackContainer'
 
-type OwnPropsType = {|
+type PropsType = {|
   route: RoutePropType<FeedbackModalRouteType>,
-  navigation: NavigationPropType<FeedbackModalRouteType>
-|}
-type DispatchPropsType = {| dispatch: Dispatch<StoreActionType> |}
-
-type InnerPropsType = {|
-  ...OwnPropsType,
-  ...DispatchPropsType,
-  cities: $ReadOnlyArray<CityModel>,
-  feedbackOrigin: FeedbackOriginType
+  navigation: NavigationPropType<OffersRouteType>
 |}
 
-type StatePropsType = StatusPropsType<InnerPropsType, OwnPropsType>
-type PropsType = {| ...OwnPropsType, ...StatePropsType, ...DispatchPropsType |}
-
-const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsType => {
-  const refreshProps = ownProps
-  if (state.cities.status === 'error') {
-    return { status: 'error', message: state.cities.message, code: state.cities.code, refreshProps }
-  }
-
-  if (state.cities.status === 'loading') {
-    return { status: 'loading', progress: 0 }
-  }
-
-  const feedbackOrigin = ownProps.route.params.isPositiveFeedback ? 'positive' : 'negative'
-  return {
-    status: 'success',
-    innerProps: {
-      ...ownProps,
-      feedbackOrigin: feedbackOrigin,
-      cities: state.cities.models
-    },
-    refreshProps
-  }
-}
-
-const mapDispatchToProps = (dispatch: Dispatch<StoreActionType>): DispatchPropsType => ({ dispatch })
-
-const refresh = (refreshProps: OwnPropsType) => {
-  const { navigation } = refreshProps
-  const navigateToFeedback = createNavigateToFeedbackModal(navigation)
-  navigateToFeedback(refreshProps.route.params)
-}
-
-const FeedbackModalContainer = (props: InnerPropsType) => {
+const FeedbackModalContainer = (props: PropsType) => {
   const { routeType, language, cityCode, isPositiveFeedback } = props.route.params
   const feedbackOrigin = isPositiveFeedback ? 'positive' : 'negative'
   return (
-    <FeedbackContainer
-      routeType={routeType}
-      feedbackOrigin={feedbackOrigin}
-      language={language}
-      cityCode={cityCode}
-      cities={props.cities}
-    />
+    <FeedbackContainer routeType={routeType} feedbackOrigin={feedbackOrigin} language={language} cityCode={cityCode} />
   )
 }
 
-export default connect<PropsType, OwnPropsType, _, _, _, _>(
-  mapStateToProps,
-  mapDispatchToProps
-)(withPayloadProvider<InnerPropsType, OwnPropsType, FeedbackModalRouteType>(refresh)(FeedbackModalContainer))
```

Comment:
Am I correct here? Without cities, we don't need `connect` and `wihtPayloadProvider`, right?

Reply:
yes thats right. We had a own ticket for it IGAPP-612